developer_guide.md

```diff
@@ -0,0 +1,55 @@
+
+## Building the Operator
+
+Create a symbolic link inside your GOPATH to the location you checked out the code
+
+    ```
+    mkdir -p ${GOPATH}/src/github.com/jlewi
+    ln -sf ${GIT_TRAINING} ${GOPATH}/src/mlkube.io
+    ```
+
+  * GIT_TRAINING should be the location where you checked out https://github.com/jlewi/mlkube.io
+
+Resolve dependencies (if you don't have glide install, check how to do it [here](https://github.com/Masterminds/glide/blob/master/README.md#install))
+
+```
+glide install
```

Comment:
Could you move the fix for the vendoring issue here?
Otherwise `go install` will fail and users might spend time trying to debug before reading further.